assets/js/components/Dashboard/NewPaMessage/SelectedStationsSummary.tsx

```diff
@@ -0,0 +1,144 @@
+import { Place } from "Models/place";
+import { Screen } from "Models/screen";
+import React, { useEffect, useState } from "react";
+import { GeoAltFill, X } from "react-bootstrap-icons";
+import fp from "lodash/fp";
+import { SILVER_LINE_ROUTES } from "Constants/constants";
+import { classWithModifier } from "../../../util";
+import pluralize from "pluralize";
+interface Props {
+  value: string[];
+  onChange: (zones: string[]) => void;
+  places: Place[];
+  busRoutes: string[];
+}
+
+const SelectedGroupTag = ({
+  numPlaces,
+  routeId,
+  onClick,
+}: {
+  numPlaces: number;
+  routeId: string;
+  onClick: () => void;
+}) => {
+  if (numPlaces === 0) return null;
+  return (
+    <div
+      className={classWithModifier("selected-group-tag", routeId.toLowerCase())}
+    >
+      {routeId}: {numPlaces} {pluralize("Station", numPlaces)}
```

Comment:
**quibble/suggestion (non-blocking):** passing `true` as the third argument to `pluralize` will prefix the string with the passed numeric value.

```suggestion
      {routeId}: {pluralize("Station", numPlaces, true)}
```

Reply:
O nice. Guess it would be good for me to read the spec of the function fully as I bring it in 😅